core/modules/filters/contains.js

```diff
@@ -0,0 +1,77 @@
+/*\
+title: $:/core/modules/filters/contains.js
+type: application/javascript
+module-type: filteroperator
+
+Filter operator for comparing the contents of an input tiddlers field and the
+operand.
+
+This filter returns the input title iff the operand is a subset of the given
+field of the input.
+
+Example 1:
+
+MyList: 1 2
+filter: [[tiddler1]][[tiddler2]]+[contains<MyList>]
+
+tiddler1 list field: 1 2 foo hello
+
+tiddler1 is returned because the list field contains both 1 and 2
+
+tiddler2 list field: 1 foo hello
+
+tiddler2 is not returned because the list field doesn't contain 2
```

Comment:
does it only work with list fields?

Reply:
It treats the field as a list, but since there is nothing to mark a field as a list field it works for any field. I used it on titles and it worked splitting the titles into separate items as expected.